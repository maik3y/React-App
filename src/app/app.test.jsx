import React from 'react';
import App from './app';
import renderer from 'react-test-renderer';

test('should render app', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
