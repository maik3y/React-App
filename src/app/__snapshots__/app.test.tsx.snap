// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render app 1`] = `"hello world"`;
